xml/admin_monitoring_alerting.xml

```diff
@@ -1071,30 +1071,26 @@ monitoring:
        <term><option>MON_DOWN</option></term>
        <listitem>
          <para>One or more monitor daemons is currently down.
-           The cluster requires a majority (more than 1/2) of
-           the monitors in order to function. When one or more
-           monitors are down, clients may have a harder time
-           forming their initial connection to the cluster as
-           they may need to try more addresses before they
-           reach an operating monitor.</para>
-         <para>The down monitor daemon should generally be
-           restarted as soon as possible to reduce the risk
-           of a subsequen monitor failure leading to a service outage.</para>
+           The cluster requires a majority of the monitors in
+           order to function. When one or more
+           monitors are down, clients have a harder time
+           forming an initial connection to the cluster.</para>
+         <para>Restart the down monitor daemon as soon as possible to reduce the risk
+           of a subsequent monitor failure.</para>
        </listitem>
      </varlistentry>
      <varlistentry>
        <term><option>MON_CLOCK_SKEW</option></term>
        <listitem>
          <para>
            The clocks on the hosts running the <systemitem class="daemon">ceph-mon</systemitem> monitor daemons are
-           not sufficiently well synchronized. This health alert is raised
+           not well synchronized. This health alert is raised
            if the cluster detects a clock skew greater than
            <option>mon_clock_drift_allowed</option>.
-           This is best resolved by synchronizing the clocks using a
-           tool like ntpd or chrony.
+           Resolve this by synchronizing the clocks using either ntpd or chrony.
```

Comment:
I think "ntpd" and "chrony" should be formatted although I am not sure if "command" or "literal" or something else is most appropriate.